src/Qir/Tools/QirTools.cs

```diff
@@ -23,7 +23,7 @@ public static class QirTools
         /// <param name="libraryDirectories">Directory where the libraries to link to are located.</param>
         /// <param name="includeDirectories">Directory where the headers needed for compilation are located.</param>
         /// <param name="executablesDirectory">Directory where the created executables are placed.</param>
-        /// <param name="debug">Enable additional debugging checks at runtime.</param>
+        /// <param name="debug">Ignored.</param>
```

Comment:
Should this be removed?

Reply:
Yeah, it should. I was hedging but I think I can make sure it is no longer called with this additional param.